lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,64 @@ def approved?
           end
         end
 
+        def merge!(options = {})
+
+          # fallback to default merge process if user "forces" merge
+          if(options[:force])
+            super options
+          else
+            options[:destination_branch] ||= 'master'
+            feature_branch_name = GitReflow.current_branch
+
+            commit_title = ask("Please enter your delivery commit title: (leaving blank will use default)")
```

Comment:
I'm not really a fan of this double prompt.  I think we'd like to see this handled in [a similar fashion to the review command](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow.rb#L53).  Ideally it would use the new `commit_message_for_merge` method provided below to populate the `"#{git_root_dir}/.git/SQUASH_MSG"` file and open it for the user to enter/edit their commit message using the `ENV['EDITOR']`.  Then once the use quits/saves we would just read that file and use the first line as the title and the rest of it for the body.